content/blog/2010/2010-03-16-slaves-building-robots.md

```diff
@@ -1,6 +1,6 @@
 ---
 :layout: post
-:title: Slaves building robots
+:title: Agents building robots
```

Comment:
For situations like this, should we rename the file and add a redirect? Or (considering the age of the article), just rename?  How bad is a broken link for a blog entry like this?

Reply:
I think just renaming suffices for now